File: client-app/src/app/models/party/FinancialSummaryDto.ts
export interface FinancialSummaryDto {
    TotalSalesInvoice: number;
    TotalPurchaseInvoice: number;
    TotalPaymentsIn: number;
    TotalPaymentsOut: number;
    TotalInvoiceNotApplied: number;
    TotalPaymentNotApplied: number;
    TotalToBePaid?: number;
    TotalToBeReceived?: number;
}

export interface InvoiceApplPaymentDto {
    InvoiceId: string;
    InvoiceTypeId: string;
    InvoiceDate: string;
    Total: number;
    AmountApplied: number;
    AmountToApply: number;
    PaymentId?: string;
    PaymentEffectiveDate?: string;
    PaymentAmount: number;
    CurrencyUomId: string;
}

export interface UnappliedInvoiceDto {
    InvoiceId: string;
    TypeDescription: string;
    InvoiceDate: string;
    Amount: number;
    UnappliedAmount: number;
    CurrencyUomId: string;
    InvoiceTypeId: string;
    InvoiceParentTypeId?: string;
}

File: Application/Accounting/Services/Models/FinancialSummaryDto.cs
namespace Application.Accounting.Services.Models;

public class FinancialSummaryDto
{
    public decimal TotalSalesInvoice { get; set; }
    public decimal TotalPurchaseInvoice { get; set; }
    public decimal TotalPaymentsIn { get; set; }
    public decimal TotalPaymentsOut { get; set; }
    public decimal TotalInvoiceNotApplied { get; set; }
    public decimal TotalPaymentNotApplied { get; set; }
    public decimal TotalToBePaid { get; set; }
    public decimal TotalToBeReceived { get; set; }
}